logbus/command.go

```diff
@@ -83,6 +85,28 @@ func (c *Command) SetStart(start time.Time) {
 	})
 }
 
+// AddDependsOn creates a delta that will be used to merge the specified target
+// ID & name into the command's list of targets on which it depends.
+func (t *Command) AddDependsOn(targetID, refName string) {
+	// Only add the dependency link once to avoid sending duplicates to Logstream.
+	t.mu.Lock()
+	if _, ok := t.dependsOn[targetID]; ok {
+		defer t.mu.Unlock()
```

Comment:
Nit: This is slightly clearer.

```suggestion
		t.mu.Unlock()
```